packages/replay-next/pages/variables.css

```diff
@@ -532,6 +725,98 @@
+  --theme-arrowpanel-color: var(--theme-popup-color);
+  --theme-arrowpanel-border-color: var(--theme-popup-border-color);
+  --theme-arrowpanel-separator: rgba(249, 249, 250, 0.1);
+  --theme-arrowpanel-dimmed: var(--theme-popup-dimmed);
+  --theme-arrowpanel-dimmed-further: rgba(249, 249, 250, 0.15);
+  --theme-arrowpanel-disabled-color: rgba(249, 249, 250, 0.5);
+
+  /* src/devtools/client/themes/splitview.css */
+  --sidemenu-selected-arrow: url("/images/item-arrow-ltr.svg");
+  --sidemenu-selected-arrow-rtl: url("/images/item-arrow-rtl.svg");
+
+  /* src/devtools/client/themes/boxmodel.css */
+  --marginbox-color: #fdffdf;
+  --borderbox-color: var(--grey-50);
+  --paddingbox-color: #e3dcff;
+  --contentbox-color: #cff0fb;
+  --marginbox-border-color: #d8e60a;
+  --contentbox-border-color: #54a9ff;
+  --position-border-color: var(--grey-50);
+
+  /* src/devtools/client/themes/rules.css */
+  --rule-highlight-background-color: var(--theme-highlight-yellow);
+  --rule-header-background-color: var(--theme-toolbar-background);
+  --rule-pseudo-class-text-color: var(--yellow-70);
+  --rule-unused-name-text-color: var(--grey-50);
+  --rule-unused-value-text-color: var(--grey-45);
+  --rule-overridden-name-text-color: var(--grey-50);
+  --rule-overridden-value-text-color: var(--grey-45);
+
+  /* src/devtools/client/themes/changes.css */
+  --diff-add-background-color: #f1feec;
+  --diff-add-text-color: #54983f;
+  --diff-remove-background-color: #fbf2f5;
+  --diff-remove-text-color: #bf7173;
+  --diff-source-background: var(--theme-toolbar-background);
+  --diff-level: 0;
+  --diff-level-offset: 10px;
+  /*
+    Minimum padding so content on the first level (zero) isn't touching the edge Added and removed lines will re-declare this to add extra padding to clear the +/- icons.
+  */
+  --diff-level-min-offset: 5px;
+
+  /* src/devtools/client/themes/computed.css */
+  --row-striped-background: rgba(247, 247, 247, 0.8);
+
+  /* src/devtools/client/themes/memory.css */
+  --cell-border-color: rgba(0, 0, 0, 0.15);
+  --cell-border-color-light: rgba(0, 0, 0, 0.1);
+  --focus-cell-border-color: rgba(0, 0, 0, 0.3);
+  --row-alt-background-color: rgba(76, 158, 217, 0.1);
+  --row-hover-background-color: rgba(76, 158, 217, 0.2);
+  --link-color: var(--blue-60);
+  --link-color-active: var(--blue-70);
+
+  /* src/devtools/client/themes/components-frame.css */
+  --frame-link-line-color: var(--theme-highlight-blue);
+  --frame-link-source: var(--theme-highlight-purple);
+
+  /* src/devtools/client/themes/widgets.css */
+  --table-splitter-color: rgba(0, 0, 0, 0.15);
+  --table-zebra-background: rgba(0, 0, 0, 0.05);
+  --sidemenu-selected-arrow: url("/images/item-arrow-ltr.svg");
+  --sidemenu-selected-arrow-rtl: url("/images/item-arrow-rtl.svg");
+
+  /* src/devtools/client/themes/perf.css */
+  --highlight-color: var(--blue-55);
+  --slider-thumb-color: var(--grey-50);
+  --slider-track-color: var(--grey-30);
+
```

Comment:
I diffed the above light and dark theme blocks.

Light theme has the following variables that dark theme doesn't:
```
--tab-standard-color
--tab-standard-hover-bgcolor
--tab-bgcolor
```

Dark theme has the following variables that light theme doens't:
```
--cm-variable
--cm-property
--cm-atom
--jellyfish-bgcolor
--timejump-text
--theme-console-input-bgcolor
--theme-toggle-handle-bgcolor
--theme-toggle-handle-bgcolor-hover
--theme-toggle-bgcolor
--theme-toggle-color
--toolbarbutton-focus-bgcolor
--toolbox-bgcolor
--unloaded-region-img
--title-hover-bgcolor
--theme-toolbar-hover
```

There shouldn't be any differences in variables defined between the themes. It either indicates that the variables aren't being used, or one of the themes is broken.

Reply:
Thank you! Yes, these are variables that over time have fallen out of sync.

My plan is rooting those out once we land these big pieces. This PR was more of a direct copy/paste, then I'll loop back around and address this over here: https://linear.app/replay/issue/DES-218/track-down-variables-that-arent-being-used-across-themes